project/proposals/119-custom-connectors.md

```diff
@@ -47,8 +47,8 @@ New API endpoints for defining custom connectors:
 | GET       | /api/**{version}**/connector-templates                           | Returns a list of known connector templates                                           |
 | GET       | /api/**{version}**/connector-templates/**{templateId}**          | Returns a specific connector template identified by the given **templateId**          |
 | POST      | /api/**{version}**/connectors/custom/info                        | Provides information like proposed name, icon and description for new connector       | 
-| POST      | /api/**{version}**/connectors/custom/                            | Create a new custom connector                                                         |
-| GET       | /api/**{version}**/connectors/custom?templateId=**{templateId}** | Lists all connectors that were created from a template identified with **templateId** |
+| POST      | /api/**{version}**/connectors/custom                             | Create a new custom connector                                                         |
+| GET       | /api/**{version}**/connectors?connectorGroupId=**{templateId}**  | Lists all connectors that were created from a template identified with **templateId** |
```

Comment:
shouldn't it be `/api/**{version}**/connectors?query=connectorGroupId=**{templateId}**` ?